examples/quickstart/quickstart.ipynb

```diff
@@ -96,8 +96,6 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.            """
</pre><p>can we add a check here to say:</p>

<pre class="ql-syntax" spellcheck="false">if len(context_str)==0:
	return "Sorry, I couldn't find an answer to your question."
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>